nin/backend/watch.js

```diff
@@ -44,7 +44,7 @@ function watch(projectPath, cb) {
                   chalk.magenta(path));
     }
 
-    cb(event, {path: path});
+    cb(event, {path: path.replace(/\\/g,"/")});
```

Comment:
Since we're changing a linting thing, we might as well add a space after the comma too :)